src/components/Router/useCrossDeviceSteps.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useState } from 'preact/hooks'
+import { StepsHook, StepsLoadingStatus } from '~types/routers'
+import { StepConfig } from '~types/steps'
+
+export const createCrossDeviceSteps = (
+  defaultSteps: StepConfig[],
+  onCompleteStep: (docData: unknown[]) => void
+): StepsHook => () => {
+  const [status, setStatus] = useState<StepsLoadingStatus>('idle')
+  const [steps] = useState(defaultSteps)
+
+  return {
+    loadNextStep: useCallback(() => setStatus('finished'), []),
+    completeStep: useCallback((data) => {
```

Comment:
Add some typing here.

Reply:
The type is inferred, the type of Data is:
`CompleteStepValue = unknown[] | Record<string, unknown>`